.github/workflows/test.yml

```diff
@@ -40,15 +45,15 @@ jobs:
     steps:
     - name: Install Go
       env:
-        GO_COMMIT: 2cfbef438049fd4c3f73d1562773ad1f93900897 # 2022-06-09
+        GO_COMMIT: 6fcbd0369d3ac9002e2b46229f90ce64ce0539e4 # 2023-01-18
       run: |
         cd $HOME
         mkdir $HOME/gotip
         cd $HOME/gotip
 
         wget -O gotip.tar.gz https://go.googlesource.com/go/+archive/${GO_COMMIT}.tar.gz
         tar -xf gotip.tar.gz
-        echo "devel go1.19-${GO_COMMIT}" >VERSION
+        echo "devel go1.20-${GO_COMMIT}" >VERSION
```

Comment:
note that master is 1.21 now :)

Reply:
Good point! In any case I've reverted this change as the tests fail with the latest commit from `master`.